tests/test_glm_synthetic_single_neuron.py

```diff
@@ -1,12 +1,12 @@
 import jax
```

Comment:
also, this script is basically the same as `test_glm_synthetic.py`. I'm thinking maybe we combine them: make a function that runs this, with an option to set the number of neurons to include. but that doesn't belong in this PR

Reply:
same as above, I kept it consistent with jax